km/km_filesys.c

```diff
@@ -93,15 +95,29 @@ static u_int64_t accept_time;   // time stamp of the latest accept HC in millise
 static int64_t accept_cnt;      // count of active accepted sockets
 
 // Account for new accept: ++ count and record the time
-static inline void km_set_accept_time(void)
+static inline void km_set_active_accept(void)
 {
    if (light_snap_accept_timeout != 0) {
       __atomic_add_fetch(&accept_cnt, 1, __ATOMIC_SEQ_CST);
-      struct timespec tp;
-      if (clock_gettime(CLOCK_MONOTONIC, &tp) != 0) {
-         km_err(2, "can't update accept time");
+      if (light_snap_accept_timeout > 0) {
+         struct timespec tp;
+         if (clock_gettime(CLOCK_MONOTONIC, &tp) != 0) {
+            km_err(2, "can't update accept time");
+            accept_time = tp.tv_sec * 1000 + tp.tv_nsec / 1000000;
```

Comment:
This statement should be outside the "if" block.
km_err() exists, so it would never be reached anyway.

Reply:
Ha! That maybe why I see no difference with or without snapshots. Good catch.